packages/key4hep-stack/package.py

```diff
@@ -267,8 +267,13 @@ class Key4hepStack(BundlePackage, Key4hepPackage):
     depends_on('py-awkward1', when="+devtools")
     depends_on('py-matplotlib', when="+devtools")
     depends_on('py-uproot4', when="+devtools")
```

Comment:
@vvolkl this will have to be changed to uproot once upstream spack has the last version right?

Reply:
No, the packagename here can stay the same, but I think the import in the python code will have to change